src/components/CardEditor/CardEditor.jsx

```diff
@@ -72,6 +72,7 @@ const propTypes = {
    */
   onValidateCardJson: PropTypes.func,
   supportedCardTypes: PropTypes.arrayOf(PropTypes.string),
+  icons: PropTypes.objectOf(PropTypes.node),
```

Comment:
this is a minor point, but since you're specifically looking for keys that match the existing DASHBOARD_EDITOR_CARD_TYPES, it would be better to use the PropTypes.shape({}) to communicate this.  At least a comment indicating that each key should match one of the card types